ios/Plugin/CapacitorHealthkit.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+@objc public class CapacitorHealthkit: NSObject {
```

Comment:
What is the purpose of this file?

Reply:
Hi,
This file is autogenerated on starting a new Capacitor 3 plugin. But the Plugin works also without it. So I removed it now.